.github/workflows/build.yml

```diff
@@ -89,7 +89,7 @@ jobs:
               with:
                   context: .
                   platforms: ${{ matrix.platforms }}
-                  push: ${{ github.ref == 'refs/heads/develop' }}
+                  push: true #${{ github.ref == 'refs/heads/develop' }}
```

Comment:
Don't forget to revert this before merging.

Reply:
Done :)